src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -99,15 +104,13 @@ public Response getUsers(@Auth AuthUser authUser, @PathParam("roleName") String
   @Produces("application/json")
   @PermitAll
   @Timed
-  public Response getUser(@Auth AuthUser authUser) {
+  public Response getUser(@Auth DuosUser duosUser) {
     try {
-      User user = userService.findUserByEmail(authUser.getEmail());
-      if (Objects.isNull(authUser.getUserStatusInfo())) {
-        samService.asyncPostRegistrationInfo(authUser);
+      if (Objects.isNull(duosUser.getUserStatusInfo())) {
+        samService.asyncPostRegistrationInfo(duosUser);
```

Comment:
Does anything in the ECM flow depend on this `asyncPostRegistrationInfo` request having been completed?

Reply:
@otchet-broad - Great question, but no, there is no connection between the Sam data vs ECM data. We're using ECM purely as a store for passport/linked account info and we use Sam to ensure our users are also Terra users,